rstream/consumer.py

```diff
@@ -200,23 +200,20 @@ async def store_offset(self, stream: str, subscriber_name: str, offset: int) ->
 
     @staticmethod
     def _filter_messages(frame: schema.Deliver, subscriber: _Subscriber) -> Iterator[bytes]:
-        if subscriber.offset_type is OffsetType.TIMESTAMP:
-            if frame.timestamp < subscriber.offset:
-                yield from ()
-            else:
-                yield from frame.get_messages()
-
-        else:
-            offset = frame.chunk_first_offset - 1
-            if subscriber.offset_type is OffsetType.NEXT:
-                offset -= 1
-
-            for message in frame.get_messages():
-                offset += 1
-                if offset >= subscriber.offset:
-                    yield message
-
-            subscriber.offset = frame.chunk_first_offset + frame.num_entries
+        minDeliverableOffset = -1
```

Comment:
I understand that this can be annoying, but python's convention is to use snake_case for var names :)

Reply:
I write go a lot, my brain is not as fast at the language switch as i'd like it to be